### PR TITLE
ref(ts): Replace organization:header hook w/ proper component hook

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by davidenwang — APPROVED** [review]
looks good, how was the hook being used before? (don't see any mention of it)

**Comment by davidenwang** [comment]
> looks good, how was the hook being used before? (don't see any mention of it)

nvm I see now

File: static/app/views/organizationContextContainer.tsx [hunk 1]
```diff
@@ -311,6 +316,7 @@ class OrganizationContextContainer extends Component<Props, State> {
         <OrganizationContext.Provider value={this.state.organization}>
```
Comment (davidenwang) [inline static/app/views/organizationContextContainer.tsx:316]: do we still need line 231-233 then?

Comment (evanpurkhiser) [reply static/app/views/organizationContextContainer.tsx:316]: Yeah migrating over the hook in getsentry

**Pull request merged**